### PR TITLE
Topic/master/news updates

### PR DESCRIPTION
**Pull request by gpaulsen**
Updates in prep for v5.0.0rc1.

### CODE REVIEW

File: HACKING.md [hunk 1]
```diff
@@ -7,6 +7,24 @@ source code form, most likely through a developer's tree (i.e., a
...
+## Obtaining Open MPI
+
+Open MPI is available from many distributions, however some users
+prefer to obtain it directly from the Open MPI community via
+prepackaged tarball (see: https://www.open-mpi.org/software/ompi/).
+The Open MPI tarball includes manpages, and openpmix and openprrte
+components, along with an auto-generated configure script.
+
+Some developers prefer to obtain Open MPI by directly cloning it
+from http://github.com/open-mpi/ompi.  It is recommended that users
```
Comment (awlauria) [inline HACKING.md:19]: https?

**Review by awlauria — APPROVED** [review]

**Comment by awlauria** [comment]
Squash and merge when ready, Thanks.

**Review by awlauria — APPROVED** [review]

**Comment by awlauria** [comment]
Last force push removed an accidental commit via a bad rebase.

**Review by awlauria — APPROVED** [review]

**Comment by ibm-ompi** [comment]
The IBM CI (PGI) build failed! Please review the log, linked below.

 Gist: https://gist.github.com/ibm-ompi/88b480fb98dfd2bdac939f023e1f8fe6

**Pull request merged**